### ISSUE TITLE
Add accessibility metadata to the test books

### ISSUE THREAD

**Issue by GeorgeKerscher** [daisy/epub-accessibility-tests#56]
We now expect accessibility metadata to be available in EPUB publications. Our test titles should meet this requirement.

**Comment by GeorgeKerscher** [comment]
Below is the metadata I think that can be cut and pasted into most test books. Some may need to be modified. If there are no page list, for example, the values should be removed.

  <metadata>

      <dc:publisher>  DAISY Consortium </dc:publisher>

  <meta>        property="dcterms:conformsTo"          id="conf">         EPUB Accessibility 1.1 - WCAG 2.2 Level AA</meta>
    
    <meta> id="certifier"> property="a11y:certifiedBy" refines="#conf"> epubtest working group </meta>

<meta property="a11y:certifierCredential">Members of the Working Group that wrote the specifications.</meta>

<meta> property="dcterms:date" refines="#certifier">2024-01-15</meta>

<meta> property="schema:structuralNavigation"</meta>

  <meta property="schema:accessibilityFeature">MathML</meta>

<meta> property="schema:accessMode">textual</meta>

    <meta> property="schema:accessMode">visual</meta>

    <meta property="schema:accessModeSufficient">textual</meta>

        <meta> property="schema:accessibilityFeature">displayTransformability</meta>
    
<meta> property="schema:accessibilityFeature">readingOrder</meta>

    <meta> property="schema:accessibilityFeature">tableOfContents</meta>

    <meta> property="schema:accessibilityFeature">unlocked</meta>

    <meta> property="schema:accessibilityFeature">pageBreakMarkers</meta>

<meta> property="schema:pageNavigation</meta>

<meta> property="schema:accessibilityFeature">alternativeText</meta>
 
  <meta property="schema:accessibilityHazard">noFlashingHazard</meta>

  <meta property="schema:accessibilityHazard">noSoundHazard</meta>

  <meta property="schema:accessibilityHazard">noMotionSimulationHazard</meta>

<meta property="schema:accessibilitySummary">
     This publication strives to conform to WCAG 2.0 Level AA.</meta>


    </metadata>

**Comment by GeorgeKerscher** [comment]
I asked Matt to review the metadata section. What is below has Matt's blessing.
<metadata>
	<dc:publisher>DAISY Consortium</dc:publisher>
	<meta property="dcterms:conformsTo" id="conf">EPUB Accessibility 1.1 - WCAG 2.2 Level AA</meta>
	<meta id="certifier" property="a11y:certifiedBy" refines="#conf">epubtest working group</meta>
	<meta property="dcterms:date" refines="#certifier">2024-01-15</meta>
	<meta property="schema:accessibilityFeature">structuralNavigation"</meta>
	<meta property="schema:accessibilityFeature">MathML</meta>
	<meta property="schema:accessMode">textual</meta>
	<meta property="schema:accessMode">visual</meta>
	<meta property="schema:accessModeSufficient">textual</meta>
	<meta property="schema:accessibilityFeature">displayTransformability</meta>
	<meta property="schema:accessibilityFeature">readingOrder</meta>
	<meta property="schema:accessibilityFeature">tableOfContents</meta>
	<meta property="schema:accessibilityFeature">unlocked</meta>
	<meta property="schema:accessibilityFeature">pageBreakMarkers</meta>
	<meta property="schema:accessibilityFeature">pageNavigation</meta>
	<meta property="schema:accessibilityFeature">alternativeText</meta>
	<meta property="schema:accessibilityHazard">noFlashingHazard</meta>
	<meta property="schema:accessibilityHazard">noSoundHazard</meta>
	<meta property="schema:accessibilityHazard">noMotionSimulationHazard</meta>
	<meta property="schema:accessibilitySummary">This publication strives to conform to WCAG 2.0 Level AA.</meta>
	
</metadata>

**Comment by marisademeglio** [comment]
Looks great George, I made a tiny edit for formatting (markup has to go in code blocks in these forms, otherwise it appears as rendered HTML with just words and no tags)

**Comment by GeorgeKerscher** [comment]
I found in the OPF a reference to the version of Schema. When looking for the proper reference for Schema, I found that it was recommended to put in the version number. This is found at:
[Schema full release ](https://schema.org/version/latest)

It says:
Note that schema.org release numbers are not generally included when you use schema.org. In contexts (e.g. related standards work) 

I therefore removed the Schema version from the metadata in the OPF.
